tests/AsyncBridge.Tests/CancelAfterTests.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+#if NET45
+using TaskEx = System.Threading.Tasks.Task;
+#endif
+
+#if NET45
+namespace ReferenceAsync.Tests
+#elif NET35
+namespace AsyncBridge.Net35.Tests
```

Comment:
@AlphaGremlin Ah, I see what happened. In master, I changed it so that net35 doesn't have the tests in a separate namespace from net40 and portable. The fix is to remove lines 12 and 13. Sorry about that!